src/ontology/mondo-ingest.Makefile

```diff
@@ -593,11 +593,34 @@ $(REPORTDIR)/%.subclass.confirmed.robot.tsv $(REPORTDIR)/%.subclass.added.robot.
 ##################################
 ## Externally managed content ####
 ##################################
-# General
-update-externally-managed-content: external-content-nord external-content-nando external-content-ordo external-content-omim
 
 EXTERNAL_CONTENT_DIR=external
 
+EXTERNAL_FILES = efo-proxy-merges \
+		mondo-clingen \
+		mondo-efo \
+		mondo-medgen \
+		mondo-omim-genes \
+		mondo-otar-subset \
+		nando-mappings \
+		nord \
+		ordo-subsets
+
+tmp/mondo-incl-external.owl: $(foreach n,$(EXTERNAL_FILES), $(EXTERNAL_CONTENT_DIR)/$(n).robot.owl) mondo.owl
+	$(ROBOT) merge $(foreach n,$^, -i $(n)) \
+		filter --term MONDO:0000001 --term MONDO:0021125 --term MONDO:0042489 --term MONDO:0021178 --select "annotations self descendants" --signature true -o $@
+
+tmp/mondo-incl-robot-report.tsv: tmp/mondo-incl-external.owl config/robot_report_external_content.txt
```

Comment:
Got: `make: *** No rule to make target 'mondo.owl', needed by 'tmp/mondo-incl-external.owl'.  Stop.`
Due to pathing issue ([comment](https://github.com/monarch-initiative/mondo-ingest/pull/628/#discussion_r1722389110))

Trying to spend a limited amount of time reviewing this so I'll skip for now.

Reply:
I think this is fixed now, a remnant from testing. Thanks for pointing that out!